portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,261 @@
+            </div>
+            <div class="col-md-4 portfolio-box wow fadeInUp">
+                <div class="portfolio-box-image">
+                    <img src="Images/data_quality.png" alt="" data-at2x="Images/data_quality.png" height="200px" width="200px">
+                </div>
+                <h3>Data Quality and Privacy analyser </h3>
+                <div class="portfolio-box-date">January 2020</div>
+                <p>Devised a platform to determine the reliability and quality of a dataset for user application.</p>
+            </div>
+        </div>
+        </div>
+</div>
+
+<!-- photo gallery -->
+<div id="photos" class="image-container">
+  <h2><center>Few Memories</center></h2>  
+  <div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+    <ol class="carousel-indicators">
+      <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+      <li data-target="#myCarousel" data-slide-to="1"></li>
+      <li data-target="#myCarousel" data-slide-to="2"></li>
+      <li data-target="#myCarousel" data-slide-to="3"></li>
+      <li data-target="#myCarousel" data-slide-to="4"></li>
+      <li data-target="#myCarousel" data-slide-to="5"></li>
+      <li data-target="#myCarousel" data-slide-to="6"></li>
+      <li data-target="#myCarousel" data-slide-to="7"></li>
+      <li data-target="#myCarousel" data-slide-to="8"></li>
+      <li data-target="#myCarousel" data-slide-to="9"></li>
+    </ol>
+
+    <!-- Adding all images -->
+
+    <div class="carousel-inner">
+       <div class="item active">
+        <img src="Images/first.jpg"alt="aa">
+    </div>
+    <div class="item">
+        <img src="Images/seventh.jpg" alt="ee">
+    </div>
+
+    <div class="item">
+        <img src="Images/second.jpg" alt="bb">
+    </div>
+    <div class="item">
+        <img src="Images/third.jpg"alt="cc">
+    </div>
+    <div class="item">
+        <img src="Images/fourth.jpg"alt="dd">
+    </div>
+    <div class="item">
+        <img src="Images/fifth.jpg" alt="ee">
+    </div>
+    <div class="item">
+        <img src="Images/sixth.jpg" alt="ee">
+    </div>
+        <div class="item">
+        <img src="Images/8.jpg" alt="ee">
+    </div>
+    <div class="item">
+        <img src="Images/9.jpg" alt="ee">
+    </div>
+    </div>
+
+    <!-- Left and right controls -->
+    <a class="left carousel-control" href="#myCarousel" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  </div>
+</div>
+
+
+
+
+<!-- footer with connection details -->
+
+
+<footer id="connect" class="footer_area p_120">
+            <div class="container">
```

Comment:
should we have consistent indentation? Here and elsewhere in file? should help with readability of the code.